docs/keyboards/ergonaut-one-s/build-guide.md

```diff
@@ -0,0 +1,164 @@
+
+![](/img/one_build_guide/pcb/bat_pins.jpg)
+
+1. Position Xiao in its designated place
+2. Start by soldering two opposite pins
+3. Solder the rest of the pins
+4. Flip the board
+5. Put some flux in the holes (marked by the red circle in the photo), and fill them with solder (you might want to heat those pins a little longer than usual)
+
+| Left                                  | Right                                 |
+| ------------------------------------- | ------------------------------------- |
+| ![](/img/one_build_guide/pcb/1_l.jpg) | ![](/img/one_build_guide/pcb/1_r.jpg) |
+| ![](/img/one_build_guide/pcb/2_l.jpg) | ![](/img/one_build_guide/pcb/2_r.jpg) |
+
+#### Diodes
+
+If you are building a "Bottom Xiao" configuration, diodes should be on the **SAME** side as Xiao.
+If you are building a "Top Xiao" configuration, diodes should be on the **OPPOSITE** side from Xiao.
+
+Please take note that diodes have polarity.
+
+![](/img/one_build_guide/pcb/diode.png)
+
+| Left                                  | Right                                 |
+| ------------------------------------- | ------------------------------------- |
+| ![](/img/one_build_guide/pcb/3_l.jpg) | ![](/img/one_build_guide/pcb/3_r.jpg) |
+
+#### Hotswap Sockets
+
+If you are building a "Bottom Xiao" configuration, sockets should be on the **SAME** side as Xiao.
+If you are building a "Top Xiao" configuration, sockets should be on the **OPPOSITE** side from Xiao.
+
+| Left                                  | Right                                 |
+| ------------------------------------- | ------------------------------------- |
+| ![](/img/one_build_guide/pcb/4_l.jpg) | ![](/img/one_build_guide/pcb/4_r.jpg) |
+
+#### Power Switches
+
+Power switches are always on the opposite side of Xiao.
+
+![](/img/one_build_guide/pcb/5.jpg)
+
+#### Test your soldering
+
+If you have a multimeter, test for shorts:
+
+1. Between "-" and "+";
+2. Between GND and 5V on the Xiao;
+3. Between GND and 3V3 on the Xiao.
+
+![](/img/one_build_guide/pcb/pins.jpg)
+
+Then connect the battery, turn on the power switch and verify the voltage between GND and 3V3 is 3.3 volts.
+If the voltage is not there, you probably need to re-solder the underside pads.
+
+If you do not have a multimeter, just proceed with the instructions.
+
+#### Batteries
+
+You can choose any side for battery wires soldering, but I suggest using the same side as Xiao.
+
+Please take note of battery polarity. Positive wire (red) should go to "+", negative wire (black) to "-".
+
+![](/img/one_build_guide/pcb/6.jpg)
+
+### Clean up
+
+Clean your PCB from flux residues. IPA (isopropyl alcohol) with Q-tips should do the trick.
+
+### Flash firmware
+
+Now it's a good time to flash firmware to a keyboard.
+
+1. Download [firmware](https://github.com/ergonautkb/one-zmk-config/releases/latest/download/ergonaut_one_firmware.zip) and unzip it.
```

Comment:
Needs to be updated when release is added to https://github.com/ergonautkb/one-s-zmk-config